user-access-public-model/src/test/java/no/unit/nva/useraccessmanagement/model/UserDtoTest.java

```diff
@@ -52,20 +49,9 @@
     private static final String FIRST_ACCESS_RIGHT = "ApproveDoi";
     private static final String SECOND_ACCESS_RIGHT = "RejectDoi";
 
-    @ParameterizedTest(name = "isValid() returns false when username is \"{0}\"")
-    @NullAndEmptySource
-    public void isValidReturnsFalseWhenUsernameIsNullOrBlank(String emptyOrNullUsername)
-        throws NoSuchMethodException, InvocationTargetException, IllegalAccessException {
-        UserDto userDto = new UserDto();
-        Method setter = UserDto.class.getDeclaredMethod("setUsername", String.class);
-        setter.setAccessible(true);
-        setter.invoke(userDto, emptyOrNullUsername);
-        assertThat(userDto.isValid(), is(equalTo(false)));
-    }
-
     @DisplayName("UserDto object contains type with value \"User\"")
     @Test
-    public void userDtoSerializedObjectContainsTypeWithValueUser() throws InvalidEntryInternalException {
+    public void userDtoSerializedObjectContainsTypeWithValueUser()throws InvalidInputException {
```

Comment:
InvlalidInputException in DTOs gives more meaning as it is an ApiGatewayException that results in BadRequest